bento_wes/runs.py

```diff
@@ -309,21 +363,14 @@ def run_list():
             "state": run["state"]
         } for run in c.fetchall()])
 
-    c.execute("SELECT r.id AS run_id, r.state AS state, rr.*, rl.* "
-              "FROM runs AS r, run_requests AS rr, run_logs AS rl "
-              "WHERE r.request = rr.id AND r.run_log = rl.id")
-
-    return jsonify([{
-        "run_id": r["run_id"],
-        "state": r["state"],
-        "details": {
-            "run_id": r["run_id"],
-            "state": r["state"],
-            "request": run_request_dict(r),
-            "run_log": run_log_dict(r["run_id"], r),
-            "task_logs": get_task_logs(c, r["run_id"])
-        }
-    } for r in c.fetchall()])
+    return jsonify(fetch_run_details(c))
```

Comment:
instead of two separate endpoints, make a query param `?public=[true|false]` like the `with_details` param above